server/src/main/kotlin/org/javacs/kt/Main.kt

```diff
@@ -15,13 +15,20 @@ class Args {
      *  - TCP Server, in which case the client has to connect to the specified tcpServerPort (used by the Docker image)
      *  - TCP Client, in which case the server will connect to the specified tcpClientPort/tcpClientHost (optionally used by VSCode)
      */
-
     @Parameter(names = ["--tcpServerPort", "-sp"])
     var tcpServerPort: Int? = null
+
     @Parameter(names = ["--tcpClientPort", "-p"])
     var tcpClientPort: Int? = null
+
     @Parameter(names = ["--tcpClientHost", "-h"])
     var tcpClientHost: String = "localhost"
+
+    @Parameter(names = ["--tcpDebug"])
```

Comment:
if open tcpDebug, the log will always on stderr, then it won't be redirected lsplog